content/en/docs/how-tos/notification.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Get notifications about a job"
+date: 2020-01-06T10:08:01-04:00
+draft: false
+---
+
+The `reporter_config` stanza in a job can be used to send notifications. [For example](https://github.com/openshift/release/blob/964a2cb316ab046a8c3d856d9c99cb40bfbbef2c/ci-operator/jobs/openshift/odo/openshift-odo-master-periodics.yaml#L2-L24), the following configuration will trigger a slack notification to channel `#odo-notifications` in [coreos.slack.com](https://coreos.slack.com/) when the job is at state `success`, `failure`, or `error`. The `report_template` defines the template of the slack message.
```

Comment:
Please just say that all notifications will go to the coreos slack, this now sounds like it was configurable, which it is not.

I don't think there is value in adding the link when we have a literal example afterwards.